pkg/controller/config/config_controller_test.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"testing"
+
+	mf "github.com/jcrossley3/manifestival"
+	op "github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestTransformManifest_WithAnnotation(t *testing.T) {
+	resourceWithAnnotation := "../../../deploy/resources/v0.9.2/addons/pipelines/s2i-go/s2i-go-build-deploy.yaml"
```

Comment:
instead of using `../../../deploy/resources/v0.9.2/addons/pipelines/s2i-go/s2i-go-build-deploy.yaml`
create a dummy file in the same location as the test or in a known test data path

the dummy file can be any kubernetes resource.

Reply:
the other files can be removed or the path can change. Hence, it is better to `unit test` with test data with fixed locations